core/quantity/src/Formatter/Format.ts

```diff
@@ -419,39 +418,38 @@ export class Format implements FormatProps {
   /**
    * Returns a JSON object that contain the specification for this Format.
    */
-  public toJson() {
-    const schemaJson: { [value: string]: any } = {};
-    schemaJson.type = Format.formatTypeToString(this.type);
-    schemaJson.precision = this.precision;
-    schemaJson.roundFactor = this.roundFactor;
-    if (undefined !== this.minWidth) schemaJson.minWidth = this.minWidth;
-    schemaJson.showSignOption = Format.showSignOptionToString(this.showSignOption);
-    schemaJson.formatTraits = Format.formatTraitsToArray(this.formatTraits);
-    schemaJson.decimalSeparator = this.decimalSeparator;
-    schemaJson.thousandSeparator = this.thousandSeparator;
-    schemaJson.uomSeparator = this.uomSeparator;
-    if (undefined !== this.scientificType) schemaJson.scientificType = Format.scientificTypeToString(this.scientificType);
-    if (undefined !== this.stationOffsetSize) schemaJson.stationOffsetSize = this.stationOffsetSize;
-    schemaJson.stationSeparator = this.stationSeparator;
-    if (undefined !== this.units) {
-      const composite: { [value: string]: any } = {};
-      composite.spacer = this.spacer;
-      composite.includeZero = this.includeZero;
-      composite.units = [];
-      this.units.forEach((unit: any) => {
-        if (undefined !== unit[1])
-          composite.units.push({
-            name: unit[0].name,
-            label: unit[1],
-          });
+  public toJson(): FormatProps {
```

Comment:
At some point we probably should rename name this to `toJSON` for consistency with how we use it in other places.  However, not needed in this PR.